lib/travis/api/app.rb

```diff
@@ -158,6 +160,15 @@ def initialize
         use Travis::Api::App::Middleware::ScopeCheck
         use Travis::Api::App::Middleware::UserAgentTracker
 
+        if ENV['OPENCENSUS_TRACING_ENABLED'] == 'true'
+          use OpenCensus::Trace::Integrations::RackMiddleware
+          OpenCensus.configure do |c|
+            c.trace.exporter = OpenCensus::Trace::Exporters::Stackdriver.new
+            c.trace.default_sampler = OpenCensus::Trace::Samplers::Probability.new
```

Comment:
this will probably need a rate argument as per https://github.com/census-instrumentation/opencensus-ruby/blob/master/lib/opencensus/trace/samplers/probability.rb#L36.